modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -57,3 +57,30 @@ The Operator is installed automatically into the `openshift-operators` namespace
 ====
 +
 . Verify that the *Status* is set to *Succeeded Up to date*  to confirm successful installation of {pipelines-title} Operator.
++
+[WARNING]
+====
+The success status may show as *Succeeded Up to date* even if installation of other components is in-progress. Therefore, it is important to verify the installation manually in the terminal.
+====
++
+. Verify the installation of all components.
++
+To verify that all components of the {pipelines-title} Operator were installed successfully, login to your cluster on the terminal and run the following commands:
++
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.9.2     True    
+$ oc get tektonpipeline,tektontrigger,tektonaddon
```

Comment:
Maybe we can add `pac` as well. `oc get tektonpipeline,tektontrigger,tektonaddon,pac`

Reply:
Done !